### ISSUE TITLE
Preguntar por 'guardar' al salir de un nuevo expediente

### ISSUE THREAD

**Issue by etchegaray** [olvidio/tramity#38]
Cuando se comienza un nuevo expediente y se rellenan los campos que se piden, al dar "guardar" pregunta si se quiere añadir un escrito o propuesta y, en caso afirmativo, se abre el formulario para añadir los datos del escrito/propuesta.

En ese formulario, si se hacen cambios en el asunto, fecha contestación, etc. y SIN GUARDAR, se pincha en añadir 'escrito', al volver de generar el escrito o la propuesta, SE HAN PERDIDO LOS CAMBIOS realizados en los datos del 'expediente' (tramite, visibilidad, vida, resolver antes de ,....). Sería bueno que antes de perder esos datos, el programa preguntara, al hacer clic en "nueva propuesta", "escrito", "plantilla", "contestar entrada" si SE DESEAN GUARDAR LOS CAMBIOS.

**Comment by Jordi59** [comment]
Sí, esto sucede con cierta frecuencia. Habría que poner este aviso de seguridad.
